src/main/charts/jira/templates/statefulset.yaml

```diff
@@ -120,13 +120,21 @@ spec:
               protocol: TCP
             {{- include "jira.additionalPorts" . | nindent 12 }}
             {{- include "common.jmx.port" . | nindent 12 }}
+          {{- if .Values.jira.readinessProbe.enabled }}
```

Comment:
Jira doesn't support liveness probe on purpose? I think we can have it here as well.

Reply:
It does https://github.com/atlassian/data-center-helm-charts/pull/626/files#diff-9ed02b7507528872403332c2e1d9880d35cf7e6e95faf7446617c633767a12c7R143

Reply:
Sorry, when I expanded the diff in the source code review, I didn't see it.